src/restaurant_inspections_api/handlers/businesses.clj

```diff
@@ -1,42 +1,37 @@
 (ns restaurant-inspections-api.handlers.businesses
   (:require [restaurant-inspections-api.validations :as validate]
             [restaurant-inspections-api.util :as util]
-            [restaurant-inspections-api.services :as srv]))
+            [restaurant-inspections-api.db :as db]
+            [clojure.string :as str]))
 
 (defn validate-businesses-params
   "Recieves all businesses query parameters (nil when not specified) and
   returns a map of valid and invalid params"
-  [zip-codes county-number per-page page]
+  [{:keys [zipCodes countyNumber perPage page]}]
```

Comment:
These vars shouldn't be camelCased. They should be zip-codes, etc. Not sure why this changed specifically :s